parlai/mturk/tasks/react_task_demo/react_custom_with_deps/run.py

```diff
@@ -10,7 +10,8 @@
     AskerOnboardingWorld, AnswererOnboardingWorld, EvaluatorOnboardingWorld, \
     MultiRoleAgentWorld
 from parlai.mturk.core.mturk_manager import MTurkManager
-from task_config import task_config
+from parlai.mturk.tasks.react_task_demo.react_custom_with_deps.task_config \
```

Comment:
since this is a demo, maybe you could leave a comment here noting why we have to use the full path to import the task_config

Reply:
Ultimately we import the rest of the things from the full path, and the fact that we haven't been doing the same for the task_config was an artifact of copypasting one of the first iterations this far. I'll note the reasoning in the docs, but I'm going to end up migrating all of our tasks over.